pypika/__init__.py

```diff
@@ -95,7 +95,7 @@
 
 __author__ = "Timothy Heys"
 __email__ = "theys@kayak.com"
-__version__ = "0.40.0"
+__version__ = "0.41.0"
```

Comment:
Hi @rob-blackbourn, would you mind reverting this as we use the bumpversion tool to manage the versioning, which also updates setup.cfg.

Reply:
@mikeengland done